spec/spec.md

```diff
@@ -109,35 +133,155 @@ resources that may exist.
+ability to make fine-grained decisions about the information that will be publicly available.
+
+## Security Considerations
+
+_This section is non-normative_
+
+### Acceptable Use
+
+_This section is non-normative_
+
+A [holder][holder] has the power to publish any verifiable credential they are holding. Before including a credential in
+a linked verifiable presentation the [holder][holder] is _RECOMMENDED_ to consider the constraints that are imposed by
+the [issuer][issuer].
+
+[Issuers][issuer] are _RECOMMENDED_ to utilize the mechanisms provided by the verifiable credentials data model
+[[spec:VC-DATA-MODEL]] to express constraints for issued credentials. For example, the properties `expirationDate` and
+`termsOfUse` can be used to control or limit the public linking of credentials.
+
+### Limiting Validity of Verifiable Presentations
+
+_This section is non-normative_
+
+Linked Verifiable Presentations are accessible and verifiable by anyone, and they might be accessible at the given URL
+indefinitely. A [holder][holder] might therefore desire to limit the validity of a presentation. This can be achieved by
+utilizing the `expires` property that is an optional element of `proof` property as specified in
+[[VC-DATA-INTEGRITY]](#ref:VC-DATA-INTEGRITY).
+
+### Verification of Linked Presentations
+
+_This section is non-normative_
+
+Unlike the interactive exchange of credentials between [holder][holder] and [verifier][verifier], linked verifiable
+presentations are statically linked and published by the [holder][holder]. Any entity that knows the [holder's][holder]
+DID is able to discover, access and verify the linked presentations without interaction between [holder][holder] and
+[verifier][verifier]. This flow requires additional caution from the [holder][holder] to ensure the integrity,
+authenticity and timeliness of linked presentations. In addition to the security mechanisms of verifiable credentials
+and presentations [[VC-DATA-INTEGRITY]](#ref:VC-DATA-INTEGRITY), the `challenge` and `domain` search parameters in the
+linked presentation URL can help the [holder][holder] to increase the level of assurance that [verifiers][verifier] will
+receive the authorized and most up to date and version of the presentation.
+
+Independent of the successful verification of linked presentations, [verifiers][verifier] should note that a linked
+verifiable presentation only proves control over the [holder's][holder] DID at the time of link creation.
+[Verifiers][verifier] SHOULD NOT grant access solely based on a linked presentation and SHOULD in addition launch a
+separate interactive request to receive proof of control over the [holder's][holder] DID. Furthermore,
+[verifiers][verifier] SHOULD ensure that the [holder's][holder] DID is referenced as subject in every verifiable
+credential to establish whether the credential applies to the [holder][holder].
```

Comment:
@peacekeeper I added a this sentence to address your concern that holders might claim credentials that aren't theirs. Does it satisfy your concern?